exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -134,15 +135,17 @@ func (prwe *prwExporter) PushMetrics(ctx context.Context, md pmetric.Metrics) er
 
 		tsMap, err := prometheusremotewrite.FromMetrics(md, prwe.exporterSettings)
 		if err != nil {
-			err = consumererror.NewPermanent(err)
+			prwe.settings.Logger.Warn("Failed to translate metrics: %s", zap.Error(err))
+			prwe.settings.Logger.Warn("Exporting remaining %s metrics.", zap.Int("converted", len(tsMap)))
```

Comment:
Are there known cases where OTEL -> Prometheus translation will fail? If so, this may be a noisy log line to emit at the warning level. Instead of logging at the `warn` level could we ensure that a metric is generated and emitted by the exporter that counts the amount of failed translations there are?

Reply:
This change will not increase the amount of error messages given that in the present state the component is already producing an error message on failed translations: the error message due to permanent error logged by the exporter helper.

please refer to the following current error message for an example of failure:
```
2022-10-19T11:27:07.375+0800    error   exporterhelper/queued_retry.go:395      Exporting failed. The error is not retryable. Dropping data.    {"kind": "exporter", "data_type": "metrics", "name": "prometheusremotewrite", "error": "Permanent error: invalid temporality and type combination;
```

I'd rather put this new feature in a different PR.